libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/ConfigurationAdaptiveDialogBot.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder.Dialogs.Adaptive;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Bot.Builder.Integration.Runtime
+{
+    internal class ConfigurationAdaptiveDialogBot : AdaptiveDialogBot
```

Comment:
Can we add a brief summary of what this class is (even if it is internal, it would help the maintainer)